locale/en.po

```diff
@@ -1408,12 +1437,29 @@ msgstr "Art by {0}"
 #: ../src/general/HelpScreen.tscn:107
 #: ../src/gui_common/dialogs/LicensesDisplay.tscn:109
 #: ../src/microbe_stage/ProcessPanel.tscn:72
-#: ../src/microbe_stage/editor/MicrobeEditor.tscn:4001
+#: ../src/microbe_stage/editor/MicrobeEditor.tscn:4072
 #: ../src/modding/ModManager.tscn:972
 #: ../src/tutorial/microbe_editor/MicrobeEditorTutorialGUI.tscn:389
 msgid "CLOSE"
 msgstr "Close"
 
+#: ../src/general/IWorldEffect.cs:77
+msgid "COMPOUND_CONCENTRATIONS_DRASTICALLY_DROPPED"
```

Comment:
When does this trigger? I don't think this is clear at all, unless literally all compound concentrations dropped by like 20%, which I don't think is ever able to happen...

Reply:
Usually happens in freebuild, initial huge population and plentiful resources, thus a big drop in compounds concentrations. I leave this here in case such scenario happens in normal game.

Reply:
> Usually happens in freebuild, initial huge population and plentiful resources, thus a big drop in compounds concentrations. I leave this here in case such scenario happens in normal game.

That's correct. It happens in Freebuild only.

Reply:
Now after reading everything, turns out that what I really should have said that this should be "GLUCOSE_CONCENTRATIONS_DRASTICALLY_DROPPED" because it's only related to glucose not all compounds.

Reply:
It's not all compounds which is why it's not plural in the translation key. In the future other compounds might also possible to get reduced significantly hence why I made the naming more general.

Reply:
So this relies on the glucose icon being next to this line? And that clarifies that this is about glucose.
I don't imagine that many other compounds follow such a nice pattern, I think most other compounds would go up and down all over the patch map without a clear direction.

Reply:
Changed this to only referring to glucose.